.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: cpp
+addons:
+  apt:
+    packages:
+    - guile-2.0-dev
+    - libgsl0-dev
+    - libfftw3-dev
+    - gfortran
+    - libhdf5-serial-dev
+    - liblapack-dev
+    - python-numpy
+    - swig
+script:
+  - ./autogen.sh --enable-shared --with-python --without-libctl
```

Comment:
`--with-libctl` will give an error simply because libctl is not installed.

See my travis file for MPB to see how to do it: https://github.com/stevengj/mpb/blob/master/.travis.yml